src/Tasks.php

```diff
@@ -1030,4 +1037,53 @@ public function migrateCleanup($opts = ['migrations' => '']) {
       return FALSE;
     }
   }
+
+
+  /**
+   * Runs a "composer update" and pushes results to the "autoupdate" branch.
+   *
+   * This command should be ran after checking out the default branch of a
+   * project. If "composer update" results in any changed files, a force push
+   * is used to ensure that the "autoupdate" branch is only one commit ahead
+   * of the default branch.
+   *
+   * @param string $profile
+   *   A specific profile to update, ex: "thinkshout/bene".
+   *
+   * @return bool
+   *   Whether or not the update succeeded.
```

Comment:
Here's the problem: Any robo commands intended to be run directly from Circle should actually return a `\Robo\Result` instead of a bool. That's something I've messed up in a lot of these custom commands. But you can see it working on HSUS:

https://github.com/thinkshout/hsus/pull/957/files

I did prove that it's a robo and not a circle problem by running the `composer update --with-dependencies` command straight in the circle `config.yml` file here: 
https://github.com/thinkshout/hsus/pull/957/commits/6abeb6ee8857f1e7e52a9806372e1fd60886d51b

and you can see that that one failed as expected:
![Screen Shot 2019-07-08 at 1 56 29 PM](https://user-images.githubusercontent.com/999525/60842109-377bdf80-a188-11e9-8933-12c627154003.png)

Reply:
That's so weird, Symfony commands let you return just a response code so I just assumed. I'll add a todo for myself to circle back and fix this.